src/components/Header/components/LocationSearch/LocationSearch.jsx

```diff
@@ -55,6 +64,9 @@ class LocationSearch extends React.Component<LocationSearchPropsType, LocationSe
             <SearchDropDown isOpenDropDown={this.state.isOpenDropDown} />
           )}
         </span>
+        <span className={classes.geolocationContainer} onClick={this.handleGetGeolocation}>
+          <IconGeolocation title="Click in order to get your correct geolocation" />
```

Comment:
Does user actually need to click something to get their geolocation? Could it be obtained automatically? Maybe I'm missing essentials of this feature.